### PR TITLE
Fix whitespace slurp and rmWhitespace in preprocessor include

### PR DESCRIPTION
**Pull request by mmis1000**
* fix whitespace slurp and rmWhitespace in preprocessor include
* add test for whitespace slurp and rmWhitespace in preprocessor include

### CODE REVIEW

**Comment by RyanZim** [comment]
@mmis1000 LGTM. Could you interactively rebase into one commit and force-push? Thanks!

**Comment by RyanZim** [comment]
@mde & @TimothyGu review?

File: test/ejs.js [hunk 1]
```diff
@@ -892,7 +892,16 @@ suite('preprocessor include', function () {
     out = ejs.render(fixture('include_preprocessor_cache.ejs'), {}, options);
     assert.equal(out, expected);
   });
-
+  
+  test('line slurp and rmWhitespace work', function() {
+    var file = 'test/fixtures/include_preprocessor_line_slurp.ejs'
+      , templete = fixture('include_preprocessor_line_slurp.ejs')
```
Comment (RyanZim) [inline test/ejs.js]: `templete` is misspelled. Should be `template`.

File: test/ejs.js [hunk 2]
```diff
@@ -892,7 +892,16 @@ suite('preprocessor include', function () {
...
+      , expected = fixture('include_preprocessor_line_slurp.html')
+      , options = {rmWhitespace: true, filename: file};
+    assert.equal(ejs.render(templete, options),
```
Comment (RyanZim) [inline test/ejs.js]: Same as above.

**Comment by mmis1000** [comment]
Fixed a few typos, and merge commits into one commit.

**Comment by RyanZim** [comment]
Thanks LGTM. I'll let @mde or @TimothyGu review this before merging.

**Comment by mde** [comment]
Thank you!
